packages/ui/src/molecules/Alert/Alert.tsx

```diff
@@ -0,0 +1,72 @@
+import type { MouseEvent, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+import type { AlertPureProps } from './AlertPure'
+import AlertPure from './AlertPure'
+import Icon from '../../atoms/Icon'
+import Button from '../../atoms/Button'
+
+export interface AlertProps extends Omit<AlertPureProps, 'onClose'> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+
+  /**
+   * A React component that will be rendered as an icon.
+   */
+  icon?: ReactNode
+
+  /**
+   * Boolean that controls the close button.
+   */
+  dismissible?: boolean
```

Comment:
I don't know if the faststore ui components should be opinionated at this level.

Reply:
You are right. I'm gonna remove this `dismissible` prop from this component. Since the component should be more generic and also the alert should not expect user interaction as per [w3 a11y docs](https://www.w3.org/TR/wai-aria-1.1/#alert). Maybe create an [AlertDialog](https://www.w3.org/TR/wai-aria-1.1/#alertdialog) in the future. @icazevedo's [comment](https://github.com/vtex/faststore/pull/1020#discussion_r756914314) made things more clear